src/crimson/osd/osd_operations/client_request.h

```diff
@@ -339,6 +339,16 @@ class ClientRequest final : public PhasedOperationT<ClientRequest>,
   };
 
   void put_historic() const;
+  static void maybe_inject_delay() {
+    if (common::local_conf()->osd_debug_inject_dispatch_delay_probability > 0) {
+      if (rand() % 10000 <
+        common::local_conf()->osd_debug_inject_dispatch_delay_probability * 10000) {
+        utime_t t;
+        t.set_from_double(common::local_conf()->osd_debug_inject_dispatch_delay_duration);
+        t.sleep();
```

Comment:
We should probably use seastar's sleep here to let the reactor run other tasks. POSIX sleep will block all other continuations on this thread.